holmes/main.py

```diff
@@ -397,53 +324,56 @@ def ask(
 
     system_prompt_rendered = load_and_render_prompt(system_prompt, template_context)  # type: ignore
 
-    if prompt_file:
+    if prompt_file and prompt:
+        raise typer.BadParameter(
+            "You cannot provide both a prompt argument and a prompt file. Please use one or the other."
+        )
+    elif prompt_file:
         if not prompt_file.is_file():
             raise typer.BadParameter(f"Prompt file not found: {prompt_file}")
         with prompt_file.open("r") as f:
             prompt = f.read()
```

Comment:
what's the user case of using the prompt file?
Should we check the emptiness of the prompt file?

Reply:
Use case is to put the prompt in a file instead of passing it on the CLI (e.g. if the instructions are long)
I think it's OK to skip the emptiness check - its the users responsibility to put meaningful content there.